ob-go.el

```diff
@@ -62,19 +62,26 @@
 ;;   package declaration.
 
 ;;; Code:
-(require 'go-mode)
 (require 'org)
 (require 'ob)
 (require 'ob-eval)
 (require 'ob-ref)
 
+(defun ob-go-get-var (params)
```

Comment:
To keep things consistent, mind naming this to org-babel-go-get-var as well as placing this after L171 with the other helper functions?